Traceback (most recent call last):
  File "/Users/guy/miniconda3/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/guy/miniconda3/lib/python3.8/site-packages/nbclient/client.py", line 1082, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/guy/miniconda3/lib/python3.8/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/guy/miniconda3/lib/python3.8/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/Users/guy/miniconda3/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/Users/guy/miniconda3/lib/python3.8/site-packages/nbclient/client.py", line 535, in async_execute
    await self.async_execute_cell(
  File "/Users/guy/miniconda3/lib/python3.8/site-packages/nbclient/client.py", line 827, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/guy/miniconda3/lib/python3.8/site-packages/nbclient/client.py", line 735, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
url = 'http://www.fdic.gov/bank/individual/failed/banklist.html'
dfs = pd.read_html(url)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mImportError[0m                               Traceback (most recent call last)
[0;32m<ipython-input-3-2fcdc94f47cc>[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0murl[0m [0;34m=[0m [0;34m'http://www.fdic.gov/bank/individual/failed/banklist.html'[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 2[0;31m [0mdfs[0m [0;34m=[0m [0mpd[0m[0;34m.[0m[0mread_html[0m[0;34m([0m[0murl[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m~/miniconda3/lib/python3.8/site-packages/pandas/util/_decorators.py[0m in [0;36mwrapper[0;34m(*args, **kwargs)[0m
[1;32m    294[0m                 )
[1;32m    295[0m                 [0mwarnings[0m[0;34m.[0m[0mwarn[0m[0;34m([0m[0mmsg[0m[0;34m,[0m [0mFutureWarning[0m[0;34m,[0m [0mstacklevel[0m[0;34m=[0m[0mstacklevel[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 296[0;31m             [0;32mreturn[0m [0mfunc[0m[0;34m([0m[0;34m*[0m[0margs[0m[0;34m,[0m [0;34m**[0m[0mkwargs[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    297[0m [0;34m[0m[0m
[1;32m    298[0m         [0;32mreturn[0m [0mwrapper[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/miniconda3/lib/python3.8/site-packages/pandas/io/html.py[0m in [0;36mread_html[0;34m(io, match, flavor, header, index_col, skiprows, attrs, parse_dates, thousands, encoding, decimal, converters, na_values, keep_default_na, displayed_only)[0m
[1;32m   1084[0m         )
[1;32m   1085[0m     [0mvalidate_header_arg[0m[0;34m([0m[0mheader[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 1086[0;31m     return _parse(
[0m[1;32m   1087[0m         [0mflavor[0m[0;34m=[0m[0mflavor[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1088[0m         [0mio[0m[0;34m=[0m[0mio[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/miniconda3/lib/python3.8/site-packages/pandas/io/html.py[0m in [0;36m_parse[0;34m(flavor, io, match, attrs, encoding, displayed_only, **kwargs)[0m
[1;32m    892[0m     [0mretained[0m [0;34m=[0m [0;32mNone[0m[0;34m[0m[0;34m[0m[0m
[1;32m    893[0m     [0;32mfor[0m [0mflav[0m [0;32min[0m [0mflavor[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 894[0;31m         [0mparser[0m [0;34m=[0m [0m_parser_dispatch[0m[0;34m([0m[0mflav[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    895[0m         [0mp[0m [0;34m=[0m [0mparser[0m[0;34m([0m[0mio[0m[0;34m,[0m [0mcompiled_match[0m[0;34m,[0m [0mattrs[0m[0;34m,[0m [0mencoding[0m[0;34m,[0m [0mdisplayed_only[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    896[0m [0;34m[0m[0m

[0;32m~/miniconda3/lib/python3.8/site-packages/pandas/io/html.py[0m in [0;36m_parser_dispatch[0;34m(flavor)[0m
[1;32m    849[0m     [0;32melse[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    850[0m         [0;32mif[0m [0;32mnot[0m [0m_HAS_LXML[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 851[0;31m             [0;32mraise[0m [0mImportError[0m[0;34m([0m[0;34m"lxml not found, please install it"[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    852[0m     [0;32mreturn[0m [0m_valid_parsers[0m[0;34m[[0m[0mflavor[0m[0;34m][0m[0;34m[0m[0;34m[0m[0m
[1;32m    853[0m [0;34m[0m[0m

[0;31mImportError[0m: lxml not found, please install it
ImportError: lxml not found, please install it

